src/hooks/useRouteAssetId/useRouteAssetId.ts

```diff
@@ -5,10 +5,17 @@ import { useEffect, useState } from 'react'
 import { matchPath, useLocation } from 'react-router'
 import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
 
+// From most to least specific
+export const assetIdPaths = ['/:chainId/:assetSubId/pool/:poolId', '/:chainId/:assetSubId']
```

Comment:
Maybe it's worthwhile to add a couple of additional comments here just to pass the context along to other developers.

```suggestion
// Make sure this array remains ordered from most to least specific to avoid early matching
export const assetIdPaths = [
    '/:chainId/:assetSubId/pool/:poolId',    // Osmosis LP token path template
    '/:chainId/:assetSubId',                 // Standard asset path template
    ]
```

Reply:
I like it. Added in https://github.com/shapeshift/web/pull/3539/commits/05cfbeee2f619e5abf3304ebeb4ffd1ca261d36c.